src/amqpvalue.c

```diff
@@ -6826,11 +6823,6 @@ static int internal_decoder_decode_bytes(INTERNAL_DECODER_DATA* internal_decoder
                                 {
                                     internal_decoder_data->decoder_state = DECODER_STATE_ERROR;
                                     LogError("Could not create inner decoder for array items");
-                                    result = MU_FAILURE;
```

Comment:
I wonder if valgrind will complain about this.

Reply:
not an issue